copy-environment/action.yml

```diff
@@ -5,11 +5,27 @@ description: "Power Platform Copy Environment Action"
 inputs:
   source-url:
     description: 'Source URL of Power Platform environment to copy; e.g. "https://test-env-source.crm.dynamics.com"'
-    required: true
+    required: false
 
   target-url:
     description: 'Target URL of Power Platform environment to copy; e.g. "https://test-env-target.crm.dynamics.com"'
-    required: true
+    required: false
+
+  source-id:
```

Comment:
same comment as w/ backup-env: let's skip introducing the -id variation of input params